Update yotpo-reviews-intro.md (#719)
magento/merchdocs: src/marketing/yotpo-reviews-intro.md

## Changes in the latest release

The most recent Magento release contains the following enhancement:

> - To enable customers to adjust when orders are synced to Yotpo, the following Yotpo extension functions have now been made programmatically accessible: getOrderCollection, setOrderCollectionLimit, and addOrderCollection.

Reviewer: ```suggestion
- The following Yotpo extension functions are now programmatically accessible: `getOrderCollection`, `setOrderCollectionLimit`, and `addOrderCollection`. This enhancement enables customers to adjust when Magento orders are synced to Yotpo.
```
